javascript/classes.json

```
@@ -526,6 +526,112 @@
           }
         }
       },
+      "private_class_fields_in": {
+        "__compat": {
+          "description": "Private class fields 'in'",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Classes/Private_class_fields",
+          "spec_url": "https://github.com/tc39/proposal-private-fields-in-in#ergonomic-brand-checks-for-private-fields",
```

This would be a valid pointer to a (rendered) spec:

```suggestion
          "spec_url": "https://tc39.es/proposal-private-fields-in-in/#sec-relational-operators",
```